File: Codechef/Dominant/Dominant.py
T = int(input())
while T:
    T -= 1
    Na, Nb, Nc = map(int, input().split())
    
    if Na > (Nb + Nc):
        print("YES")
    
    elif Nb > (Na + Nc):
        print("YES")
    
    elif Nc > (Na + Nb):
        print("YES")

    
    else:
        print("NO")

File: Codechef/Dominant/Dominant.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while(T--)
    {
        int Na, Nb, Nc;
        cin >> Na >> Nb >> Nc;

        if (Na > (Nb + Nc))
        {
            cout << "YES" << endl;
        }
        else if (Nb > (Na + Nc))
        {
            cout << "YES" << endl;
        }
        else if (Nc > (Na + Nb))
        {
            cout << "YES" << endl;
        }
        else
        {
            cout << "NO" << endl;
        }
    }

    return 0;
}